pkgs/development/python-modules/langchain-chroma/default.nix

```diff
@@ -2,6 +2,7 @@
   lib,
   buildPythonPackage,
   fetchFromGitHub,
+  chroma-hnswlib,
```

Comment:
We need to use this somewhere, too.

Reply:
This is not needed in the end.
I have just removed it.